packages/web-app-files/src/services/folder.ts

```diff
@@ -12,7 +12,7 @@ import {
   FolderLoaderSharedWithMe,
   FolderLoaderSharedWithOthers,
   FolderLoaderTrashbin
-} from './folder/'
+} from './folder/index'
```

Comment:
Huh? So the appended `/` doesn't help to differentiate  folder `folder` and file `folder.ts`?

Reply:
I'm not sure why I had to add this - needs double checking